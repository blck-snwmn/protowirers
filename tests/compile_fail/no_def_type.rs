use protowirers::*;

#[derive(Proto)]
struct Sample {
    #[def(field_num = 1, field_num = 1)]
    s: u32,
}

fn main() {}
