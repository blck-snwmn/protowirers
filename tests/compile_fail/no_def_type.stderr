error: def_type is required in #[def(...)]
 --> $DIR/no_def_type.rs:5:5
  |
5 |     #[def(field_num = 1)]
  |     ^^^^^^^^^^^^^^^^^^^^^
